NDTensors/ext/NDTensorsCUDAExt/typeparameteraccessors.jl

```diff
@@ -0,0 +1,20 @@
+# # `TypeParameterAccessors.jl` overloads.
+using NDTensors.TypeParameterAccessors: TypeParameterAccessors, Position, parameter
+## TODO this seems like a `GPUArrays` generic function
+storagemode(t::Type{<:CuArray}) = parameter(t, 3)
+function TypeParameterAccessors.position(::Type{<:CuArray}, ::typeof(storagemode))
+  return Position(3)
+end
+
+function TypeParameterAccessors.default_parameter(::Type{<:CuArray}, ::typeof(eltype))
+  return Float64
+end
+TypeParameterAccessors.default_parameter(::Type{<:CuArray}, ::typeof(ndims)) = 1
```

Comment:
Seems like these could be generic `AbstractArray` definitions.

Reply:
Yes they can, I removed these from the code, thank!